content/docs/intro/concepts/state.md

```diff
@@ -13,124 +13,81 @@ menu:
 aliases: ["/docs/reference/state/"]
 ---
 
-## State
+Pulumi stores metadata about your infrastructure so that it can manage your cloud resources. This metadata is called _state_ and is how Pulumi knows when and how to create, read, delete, or update cloud resources. Pulumi stores this metadata, the state, in a _backend_ of your choosing. Options include the Pulumi Service, a secure and reliable hosted application with policies and safeguards to facilitate team collaboration, in addition to object storage in AWS, Azure, GCP, or a simple filesystem. Most of the time, you don't need to explicitly worry about state and backends, but in some cases you may.
```

Comment:
AWS also means AWS compatible - e.g. Minio / DigitalOcean / Ceph etc

Reply:
Does our current S3 backend support those? I thought `s3://` automatically assumes the bucket is in AWS. If it does, great, I will definitely change the text here and would like to add even more detail on how to configure this.

Reply:
yeah we get it for free from https://gocloud.dev/howto/blob/#s3-compatible :)

Reply:
Nice!!! I will add your suggested changes as well as add a little note about that in the S3 (compatible) backend section.